app/components/dex-preview.jsx

```diff
@@ -4,9 +4,20 @@ import { connect } from 'react-redux';
 import { ProgressComponent } from './progress';
 
 export function DexPreview ({ dex, username }) {
+  let dexIndicator = null;
+
+  if (dex.shiny) {
+    dexIndicator = (
+      <div className="dex-indicator">
+        <i className="fa fa-star" />
```

Comment:
thoughts on having a `title` attr just in case someone is new to it all and wants to know what the red star means?